source/development/advanced/reference.rst

```diff
@@ -1971,6 +1973,14 @@ database server is running and port\_2 is its port number. All access to
 database will automatically switch from one server to another one in the
 list if the one which was used has died.
 
+Finally, if you want to define a TANGO\_HOST variable that explicitly 
+does not access the Tango database, you can add a \#dbase=no suffix:
+
+TANGO\_HOST=<host\_1>:<port\_1>\#dbase=no
+
+In this case, the host\_1 and port\_1 must refer to the device server 
+you are addressing.
+
```

Comment:
Thanks for adding this @NicholasRees, but it is a little premature.  Maybe it wasn't clear in my [PyTango Status Update](https://ajoubertza.github.io/pytango-status-updates/#16) today, but this is a feature that still needs to be implemented.  The main benefit would be for automated unit/component testing where we are running without a Tango database.

It is quite easy to do in PyTango, so is likely to happen there first.  I'm not sure if it will be useful in cppTango, or JTango?  Maybe the maintainers could comment?

Reply:
I understood that it was supported in C++, but not python, but I was clearly wrong. There clearly need to be some code mod's in a number of places to support this, so this is a suggestion, not even a firm proposal. If we are going to change the syntax of TANGO_HOST we clearly need to think about it. I would like it to support a search list of hosts with different  Tango databases, for example.